pkgs/development/python-modules/dirsearch/default.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  python,
+  # deps
+  beautifulsoup4,
+  certifi,
+  cffi,
+  chardet,
+  charset-normalizer,
+  colorama,
+  cryptography,
+  defusedxml,
+  idna,
+  jinja2,
+  markupsafe,
+  pyopenssl,
+  pyparsing,
+  pysocks,
+  pyspnego,
+  requests,
+  requests-ntlm,
+  setuptools,
+  urllib3,
+}:
+
+buildPythonPackage rec {
+  pname = "dirsearch";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "maurosoria";
+    repo = "dirsearch";
+    rev = "v${version}";
+    hash = "sha256-eXB103qUB3m7V/9hlq2xv3Y3bIz89/pGJsbPZQ+AZXs=";
+  };
+
+  # setup.py does some weird stuff with mktemp
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail 'os.chdir(env_dir)' "" \
+      --replace-fail 'shutil.copytree(os.path.abspath(os.getcwd()), os.path.join(env_dir, "dirsearch"))' ""
+  '';
+
+  dependencies = [
+    beautifulsoup4
+    certifi
+    cffi
+    chardet
+    charset-normalizer
+    colorama
+    cryptography
+    defusedxml
+    idna
+    jinja2
+    markupsafe
+    pyopenssl
+    pyparsing
+    pysocks
+    pyspnego
```

Comment:
I couldn't find a reference of this dependency upstream

Reply:
It added as a replacement for `ntlm-auth`, because nixpkgs told me that `ntlm-auth` has been removed, and I should use `pyspnego`. `pyspnego` seems not to be a drop-in replacement tho, and the programs seems to work without it.